bookserver/routers/rslogging.py

```diff
@@ -101,20 +100,32 @@ async def log_book_event(entry: LogItemIncoming, request: Request):
     useinfo_entry = UseinfoValidation(**useinfo_dict)
     rslogger.debug(useinfo_entry)
     idx = await create_useinfo_entry(useinfo_entry)
-
-    # Always provide a server-supplied timestamp to the client.
     response_dict = dict(timestamp=entry.timestamp)
-    event = entry.event
-    if event in EVENT2TABLE:
-        rcd = runestone_component_dict[EVENT2TABLE[event]]
-        # Validate the fields for this component.
+    if entry.event in EVENT2TABLE:
+        rcd = runestone_component_dict[EVENT2TABLE[entry.event]]
+        if entry.event == "unittest":
+            # info we need looks like: "act":"percent:100.0:passed:2:failed:0"
+            ppf = entry.act.split(":")
+            entry.passed = int(ppf[3])
```

Comment:
Need to handle problems if these values don't parse -- return a 404 instead of just raising an exception.

Reply:
I guess, but this doesn't currently happen as there is only one piece of code that generates these messages and it does so correctly.

Reply:
Sounds good. Would you mind adding a link to the code that generates these values in the docs?